tcms/testcases/tests/test_views.py

```diff
@@ -100,6 +100,8 @@ def setUpTestData(cls):
             "category": cls.case.category.pk,
             "case_status": cls.case_status_confirmed.pk,
             "priority": cls.case.priority.pk,
+            "setup_duration": cls.case.setup_duration,
+            "testing_duration": cls.case.testing_duration,
```

Comment:
These are probably None by default so the change doesn't make much sense. Make sure that you are passing valid durations and then assert that they have been indeed stored into the database.